src/DynamoCore/Search/Legacy/SearchElements/SearchElementBase.cs

```diff
@@ -137,50 +137,34 @@ public class LibraryItem
         public IEnumerable<PortInfo> ReturnKeys { get; private set; }
 
 
-        public LibraryItem(SearchElementBase node, DynamoModel dynamoModel)
+        public LibraryItem(NodeSearchElement node, DynamoModel dynamoModel)
         {
             Category = node.FullCategoryName;
-            Type = node.Type;
             DisplayName = Name = node.Name;
             CreationName = node.CreationName;
             Description = node.Description;
-            Searchable = node.Searchable;
```

Comment:
`NodeSearchElement` doesn't contain `Type` and `Searchable` fields anymore. Also they are unused in Flood. So just remove them